Gopkg.lock

```diff
@@ -15,7 +15,6 @@
   version = "1.1"
 
 [[projects]]
-  branch = "master"
```

Comment:
@mknister why did you remove?

Reply:
@smtakeda this is something that Goland did automatically and I assumed it was on purpose. Should I revert this change?

Reply:
@ChTimTsubasa do you know about this line?

Reply:
This file is generated by dep. If dep changed it the we should respect its change.